community-plugins.json

```diff
@@ -2509,5 +2509,12 @@
         "author": "Robin Haupt",
         "repo": "Robin-Haupt-1/Obsidian-Map-of-Content",
         "branch": "main"
+    },
+    {
+        "id": "highlightr-plugin",
+        "name": "Highlightr",
+        "description": "Highlightr is a simple plugin that brings a minimal and aesthetically pleasing highlighting menu into the Obsidian note-taking app. This plugin makes color-coded highlighting much easier with a user-friendly assortment of highlight colors 🎨.",
```

Comment:
I recommend making the description a bit shorter since this is what users will see in the plugin gallery.

Maybe something like?

> A minimal and aesthetically pleasing highlighting menu that makes color-coded highlighting much easier with a configurable assortment of highlight colors 🎨.